resources/css/style.css

```diff
@@ -0,0 +1,103 @@
+/* stylelint-disable function-url-quotes */
+/* stylelint-disable font-family-name-quotes */
+
+/* Global */
+
+html {
+  font-size: 62.5%; /* 1 REM = 10PX */
```

Comment:
Commented code makes your files look nasty, unorganised, and unprofessional.

- [ ] Please, remove the commented code from your files for the reasons mentions above :+1: